src/pretalx/static/agenda/scss/_speaker.scss

```diff
@@ -18,30 +18,31 @@
       border-bottom: 1px solid rgba(0, 0, 0, 0.1);
     }
   }
+}
 
-  .speakers {
-    border: 1px solid #4CAF50;
-    border-top: 4px solid #4CAF50;
-    background-color: rgba(255, 255, 255, 0.76);
-    box-sizing: border-box;
-    color: rgba(0, 0, 0, 0.87);
-    display: block;
-    padding: 5px 10px;
+/* used for speaker list on talk page as well as talk list on speaker page */
+.bordered-list {
```

Comment:
In case you come up with a nicer name for this class, let me know. Maybe I can think of one tomorrow.